<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-02-18T18:42:19 IST" name="Suite" finished-at="2025-02-18T18:42:24 IST" duration-ms="4674">
    <groups>
      <group name="smoke">
        <method signature="LoginTest.verifyWhetherUserIsAbleLoginSuccessfullywithValidUsernameandValidPassword()[pri:0, instance:testscript.LoginTest@28cda624]" name="verifyWhetherUserIsAbleLoginSuccessfullywithValidUsernameandValidPassword" class="testscript.LoginTest"/>
      </group> <!-- smoke -->
    </groups>
    <test started-at="2025-02-18T18:42:19 IST" name="Test" finished-at="2025-02-18T18:42:24 IST" duration-ms="4674">
      <class name="testscript.LoginTest">
        <test-method signature="verifyWhetherUserIsAbleLoginSuccessfullywithValidUsernameandValidPassword()[pri:0, instance:testscript.LoginTest@28cda624]" started-at="2025-02-18T18:42:19 IST" name="verifyWhetherUserIsAbleLoginSuccessfullywithValidUsernameandValidPassword" description="Verify Whether User is able to Login Successfully with Valid Username and Valid Password" finished-at="2025-02-18T18:42:24 IST" duration-ms="4630" status="FAIL">
          <exception class="java.lang.ExceptionInInitializerError">
            <full-stacktrace>
              <![CDATA[java.lang.ExceptionInInitializerError
at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized0(Native Method)
at java.base/jdk.internal.misc.Unsafe.ensureClassInitialized(Unsafe.java:1160)
at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.ensureClassInitialized(MethodHandleAccessorFactory.java:300)
at java.base/jdk.internal.reflect.MethodHandleAccessorFactory.newConstructorAccessor(MethodHandleAccessorFactory.java:103)
at java.base/jdk.internal.reflect.ReflectionFactory.newConstructorAccessor(ReflectionFactory.java:200)
at java.base/java.lang.reflect.Constructor.acquireConstructorAccessor(Constructor.java:549)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
at org.testng.internal.objects.InstanceCreator.newInstance(InstanceCreator.java:38)
at org.testng.internal.objects.InstanceCreator.newInstance(InstanceCreator.java:59)
at org.testng.ITestObjectFactory.newInstance(ITestObjectFactory.java:10)
at org.testng.internal.objects.SimpleObjectDispenser.dispense(SimpleObjectDispenser.java:60)
at org.testng.internal.objects.GuiceBasedObjectDispenser.dispense(GuiceBasedObjectDispenser.java:28)
at org.testng.internal.BaseTestMethod.computeRetryAnalyzerInstanceToUse(BaseTestMethod.java:843)
at org.testng.internal.BaseTestMethod.getRetryAnalyzerConsideringMethodParameters(BaseTestMethod.java:817)
at org.testng.internal.BaseTestMethod.getRetryAnalyzer(BaseTestMethod.java:681)
at org.testng.internal.invokers.TestInvoker.shouldRetryTestMethod(TestInvoker.java:549)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:726)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.ClassCastException: class org.apache.logging.slf4j.SLF4JLogger cannot be cast to class freemarker.log.Logger (org.apache.logging.slf4j.SLF4JLogger and freemarker.log.Logger are in unnamed module of loader 'app')
at retry.Retry.<clinit>(Retry.java:12)
... 40 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ExceptionInInitializerError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyWhetherUserIsAbleLoginSuccessfullywithValidUsernameandValidPassword -->
        <test-method is-config="true" signature="initialiseBrowser(java.lang.String)[pri:0, instance:testscript.LoginTest@28cda624]" started-at="2025-02-18T18:42:19 IST" name="initialiseBrowser" finished-at="2025-02-18T18:42:22 IST" duration-ms="2776" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialiseBrowser -->
      </class> <!-- testscript.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
